src/xr_3da/entry_point.cpp

```diff
@@ -10,6 +13,7 @@
 #endif
 #include "xrEngine/main.h"
 #include "xrEngine/splash.h"
+#include "xrCore/command_line_key.h"
```

Comment:
Just include this include into `xrCore.h` and then we don't need to include it everywhere anymore

Reply:
Is it ok if after this I'll have to include xrCore.h in the source files it wasn't included before?

Reply:
Hm, what file?

Reply:
LocatorAPI.cpp
LocatorAPI_auth.cpp
log.cpp
xrDebug.cpp
xrMemory.cpp

Reply:
xrCore.h is already included in stdafx.h, which is included to these files. There shouldn't be any problems with this...

Reply:
In that case I'll just include it in xrCore.h